src/components/modals/EntityCreatorEditor/EntityCreatorComponent.tsx

```diff
@@ -351,6 +386,14 @@ const Component: React.SFC<Props> = (props) => {
                 <FormattedMessageId id={FM.ENTITYCREATOREDITOR_DELETE_ENUM_ERROR_WARNING} />
             </div>}
         />
+        <ConditionCreatorModal
```

Comment:
I suppose I should rename this `ConditionModal` since it's not constrained to creating and should handle everything about conditions.

Reply:
ConditionEditorModal ?

Reply:
I don't think that's better, still restricts it to editing even though is primarily used for creating. I don't want people to think it's only for editing or only for creating and don't want to go down the path we have with other modals where we add CreatorEditor nonsense. I thought by dropping both terms it would imply it can do various things with Conditions

Reply:
Maybe it's just me, but I consider creation and editing to be fungible.

Agree that "CreatorEditor" is not a good option.